README.md

```diff
@@ -11,4 +10,142 @@ UAVCAN is an open lightweight protocol designed for reliable communication in ae
 robust vehicle bus networks.
 
 * [**UAVCAN website**](http://uavcan.org)
-* [**UAVCAN mailing list**](https://groups.google.com/forum/#!forum/uavcan)
+* [**UAVCAN forum**](https://forum.uavcan.org)
+
+## Identifier ranges
+
+### Subjects
+
+For message subjects, the following range mapping is adopted (limits inclusive).
+Unused ranges are reserved for future expansion of adjacent ranges.
+
+From    | To        | Purpose
+--------|-----------|------------------------------------------------
+0       | 0         | Not a valid ID
+1       | 32767     | Dynamic
+61440   | 62803     | Vendor-specific public static
+62804   | 65535     | Standard static
```

Comment:
How does this work with anonymous transfers? They need low subject IDs to work. Maybe 0-15 should be reserved as well for high priority static protocol transfers?

Reply:
This is due to another bit of hidden reasoning I didn't share with the world yet, sorry. I should have managed my edits more carefully to avoid unnecessary confusion, I was just trying to take shortcuts to save time.

My idea was to invert the subject ID used in anonymous message transfers, moving them from [0, 7] to [65528, 65535]. That doesn't require us to split the ID range.

Reply:
I don't like the "invert by default" approach, it's surprising behavior. We should also allow application designers to design data types usable with anonymous transfers as well. This means that we should also have some usable messages in the low ID range. What about one of the following things:

 - Reserve some IDs in the low ID range. That will give us the additional benefit of having some high priority IDs available.
 - Add an `invert` bit to the anonymous transfers making both the low ID and high ID scale usable. bit23 looks like a nice enough candidate.

Reply:
> Add an `invert` bit to the anonymous transfers making both the low ID and high ID scale usable. bit23 looks like a nice enough candidate.

I like this. It would essentially define which side of the range the short ID should cling to: zero or 65535. The bit 23 seems to be situated in the middle of a reserved field, so I would rather use either 21 or 25, otherwise it seems wasteful.

Reply:
Great! We should move this discussion to somewhere related to specification. The bit 23 was rooted in that we might want to increase number of bit usable in anonymous transfers at a later point? Anyway, let's take this to a different issue/thread.